### PR TITLE
Initial addition of mojaloop-sim chart

### PR DESCRIPTION
**Pull request by ksatya77**

### CODE REVIEW

File: mojaloop-simulator/templates/_helpers.tpl [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+{{- else -}}
+{{- $name := default .Chart.Name .Values.nameOverride -}}
+{{- if contains $name .Release.Name -}}
+{{- .Release.Name | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+{{- end -}}
+{{- end -}}
+
```
Comment (msk-) [inline mojaloop-simulator/templates/_helpers.tpl:26]:
```suggestion
```
Unused.

**Comment by msk-** [comment]
It's possible that template-specialised resource names will exceed 63 chars. But that should probably be addressed in future work as this chart is working for a couple of users now. You could perhaps add this to the gigantic TODO list in the values file. My favourite organisational technique.

**Comment by rmothilal** [comment]
@ksatya77 the branch is out of sync with master

File: mojaloop-simulator/values.yaml [hunk 2]
```diff
@@ -0,0 +1,430 @@
...
+# a payerfsp without a cache and a payeefsp with a cache.
+#
+# simulators:
+#   payerfsp:
+#     config:
+#       cache:
+#         enabled: false
+#   payeefsp: {}
+
+# TODO & notes:
```
Comment (elnyry) [inline mojaloop-simulator/values.yaml:25]: Is this todo and notes section needed?

**Comment by ksatya77** [comment]
@rmothilal .... I do not see any merge conflicts so once this branch is just an addition to latest master...but to make sure, I also got the latest master and merged into this PR branch...are you still seeing outdated branch??

**Comment by msk-** [comment]
@ksatya77 sorry for confusion- I saw Rajiv's comment and merged from master.

**Comment by ksatya77** [comment]
@msk- ...even better..thx much..

File: mojaloop-simulator/values.yaml (hunk 2 above)
Comment (ksatya77) [reply mojaloop-simulator/values.yaml:25]: probably server as a good reminder for future tasks...you concerned that this being public, y'ld rather prefer them as stories?

Comment (elnyry) [reply mojaloop-simulator/values.yaml:25]: ok.. yeah probably.. I don't think this place has much visibility.. But I'll resolve this for now.

**Comment by elnyry** [comment]
@ksatya77 is the mojaloop-simulator not supposed to be deployed with 'mojaloop' charts? if thats the intention, then its fine.. (asking this as it wasn't included under /mojaloop/requirements.yaml , etc)

**Comment by ksatya77** [comment]
yes @elnyry ...depending on the deployments, for ex labs, we do not want these sims to be part of mojaloop chart...cos these go to an entirely different cluster than mojaloop

**Comment by ksatya77** [comment]
@elnyry @rmothilal ...if you guys are satisfied with the comments, can one of you please give your approval...as you are the codeowners..

**Pull request merged**